autoload/airline/extensions.vim

```diff
@@ -310,6 +310,11 @@ function! airline#extensions#load()
     call add(loaded_ext, 'obsession')
   endif
 
+  if (get(g:, 'airline#extensions#vimtex#enabled', 1))
```

Comment:
There should be another condition, that actually checks, whether vimtex is installed